targets/hasura/migrations/1613478848457_add_audit/up.sql

```diff
@@ -0,0 +1,35 @@
+--
+-- Audit the following table using audit triggers
+--
+
+select audit.audit_table('auth.refresh_tokens');
+select audit.audit_table('auth.user_roles');
+select audit.audit_table('auth.users', 'true', 'true', '{password,secret_token}');
+select audit.audit_table('alert_notes');
+select audit.audit_table('alert_status');
+select audit.audit_table('alerts', 'false');
+select audit.audit_table('document_relations', 'false');
```

Comment:
Doesn't work, `audit_table` with 2 arguments doesn't exist.
```suggestion
select audit.audit_table('document_relations', 'false', 'true');
```

Reply:
or add the function with 2 parameters :)